configuration/product_variant/lab_tests.csv

```diff
@@ -0,0 +1,80 @@
+id,product_variant_ids/uuid,name,product_variant_ids/categ_id/id,type,lst_price

```

Comment:
Rename the headers as per the specs:
`uuid` and `categ_id/id`

Reply:
Done